### PR TITLE
C# SDK change with Hub's certificate and operation tasks

### PR DESCRIPTION
**Pull request by anusapan**
## Description
Regenerated the SDK based off a new swagger and add/update the test cases for updated SDK.
[Updated swagger PR](https://github.com/Azure/azure-rest-api-specs/commit/9bc154a142ae39c3b900ba91043aabed1ef8e2f3)

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: NuGet.Config [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-     <!-- To enable LocalFeed for testing uncomment the following line -->
-     <add key="NugetOfficialV3" value="https://api.nuget.org/v3/index.json" />
-     <add key="Local" value="tools/LocalNugetFeed" />
-   </packageSources>
+	<!-- To enable LocalFeed for testing uncomment the following line -->
```
Comment (shahabhijeet) [inline NuGet.Config]: @anusapan what is the need to edit this file? All this is doing is deleting spaces. Can you revert this change.

File: src/SDKs/IotHub/IotHub.Tests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -35,5 +35,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.1")]
-[assembly: AssemblyFileVersion("1.0.0.1")]
+[assembly: AssemblyVersion("1.0.0.2")]
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/IotHub.Tests/Properties/AssemblyInfo.cs]: @anusapan AssemblyVersion only changes when you update the Major version. Please do not update AssemblyVersion and make it same as AssemblyFileVersion

File: src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj [hunk 3]
```diff
@@ -5,10 +5,13 @@
...
     <AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-    <VersionPrefix>1.1.2</VersionPrefix>
+    <VersionPrefix>1.1.3</VersionPrefix>
     <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj]: @anusapan you do not need this property anymore, so please remove it. If you get in the upstream changes, this will no longer be needed.

File: src/SDKs/IotHub/IotHub.Tests/IotHub.Tests.csproj [hunk 4]
```diff
@@ -4,13 +4,17 @@
...
   </PropertyGroup>
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/IotHub.Tests/IotHub.Tests.csproj]: @anusapan please use latest version of Resource manager to record all your tests. You are still using 1.1.0, latest version is 1.6.0
YOU will have add reference to the new nuget, use the namespace from the new nuget and remove all traces of old namespace and then record your tests.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: NuGet.Config (hunk 1 above)
Comment (anusapan) [reply NuGet.Config]: reverted

File: src/SDKs/IotHub/IotHub.Tests/IotHub.Tests.csproj (hunk 4 above)
Comment (anusapan) [reply src/SDKs/IotHub/IotHub.Tests/IotHub.Tests.csproj]: added new references of ResourceManager and recorded test again

File: src/SDKs/IotHub/IotHub.Tests/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (anusapan) [reply src/SDKs/IotHub/IotHub.Tests/Properties/AssemblyInfo.cs]: reverted

File: src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj (hunk 3 above)
Comment (anusapan) [reply src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj]: reverted

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

File: src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj [hunk 5]
```diff
@@ -5,7 +5,7 @@
     <Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
     <AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-    <VersionPrefix>1.1.2</VersionPrefix>
+    <VersionPrefix>1.1.3</VersionPrefix>
     <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj]: @anusapan can you add package release notes like below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj (hunk 5 above)
Comment (anusapan) [reply src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj]: added PackageReleaseNotes.

**Comment by shahabhijeet** [comment]
@azuresdkci retest this please

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Review by shahabhijeet — APPROVED** [review]
